charts/celeborn/templates/configmap.yaml

```diff
@@ -18,7 +18,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: celeborn-conf
+  name: {{ .Values.celebornClusterPrefix }}-conf
```

Comment:
The idea looks good, I'm concern with the name, maybe `celebornResourcePrefix`.

Another idea is to use helm release name as we discussed offline, but it requires to do investigation whether it's designed/suitable for such cases before applying.

Reply:
Related to [Helm.sh#Release](https://helm.sh/docs/glossary/#release) described, `Release` suits our scene.

>Release
When a chart is installed, the Helm library creates a release to track that installation.
A single chart may be installed many times into the same cluster, and create many different releases. For example, one can install three PostgreSQL databases by running helm install three times with a different release name.

Instead of adding a new var in `values.yml`, we can use the built-in `{ .Release.name }`

Reply:
SGTM, thanks for investigating.